### PR TITLE
adds a brain damage line

### PR DESCRIPTION
**Pull request by Iamgoofball**
# IM GONNA CUM

### CODE REVIEW

**Comment by Errorin** [comment]
# **BIG CUMS**

**Comment by The-clown-known-as-Pepe** [comment]
💦

**Comment by Bluespace** [comment]
Add another that says 
"robotacist has FLASH!!! REVV?"

**Comment by Repukan** [comment]
Add another one that says 

"Year of the goof"

**Comment by AnonymousNow** [comment]
Can't like on principle, since "revert Goonchem" is a line, for some reason.

**Comment by GunHog** [comment]
"I test this"?

File: strings/brain_damage_lines.json [hunk 1]
```diff
@@ -63,6 +63,7 @@
         "@pick(semicolon)wtf??????????? @pick(bug)",
         "@pick(semicolon)i ran into the supermattre ten i dsappeard @pick(bug)",
         "DON'T EVER TUCH ME",
+		"@pick(semicolon)ay-tee ess-tees AY-TEE ESS-TEES!!!"
```
Comment (Cyberboss) [inline strings/brain_damage_lines.json]: indentation

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by Bawhoppen** [comment]
NO MORE BRAIN DAMAGE LINES

**Comment by KorPhaeron** [comment]
2 days to fix an indentation is stale

**Comment by Iamgoofball** [comment]
Lmao my comment didn't go through

**Pull request closed without merging**